remote/remote.go

```diff
@@ -391,7 +391,7 @@ func getCleanWpCliArgumentArray(wpCliCmdString string) ([]string, error) {
 
 	// Remove quotes from the args
 	for i := range cleanArgs {
-		if( ! isJSON( cleanArgs[i])) { //don't alter JSON arguments
```

Comment:
I've just checked the `isJSON` implementation and I was wondering if would be better to use `json.Valid`.

So, instead of:
```go
func isJSON(str string) bool {
	var js json.RawMessage
	return json.Unmarshal([]byte(str), &js) == nil
}
```

we would have:
```go
func isJSON(str string) bool {
	return json.Valid([]byte(str))
}
```

What do you think?

Reply:
Good idea, will replace it, thanks!